elements/pfe-card/demo/index.html

```diff
@@ -162,7 +162,7 @@ <h2>Imaged card with no overflow</h2>
       </pfe-card>
 
       <pfe-card pfe-color="lightest">
```

Comment:
Should it be `pfe-c-color`?

Reply:
Yes, it should. I'll make the update to card and band.